### DISCUSSION TITLE
how to change the character for this vertical line?

### DISCUSSION THREAD

**Discussion by hopezh** [shellRaining/hlchunk.nvim#98] [Q&A] [answered]
As shown below, how to change the character for the **vertical line** indicating the nearest context?

![2024-04-17 23 37 12](https://github.com/shellRaining/hlchunk.nvim/assets/23435810/5c94d442-3e56-466e-bf11-a34b2ba2c1b5)

**Comment by shellRaining** [comment, marked as answer]
this is a bug and fixed in latest commit
